src/cursor.rs

```diff
@@ -132,6 +132,25 @@ impl Command for MoveToColumn {
     }
 }
 
+/// A command that moves the terminal cursor to the given row on the current column.
+///
+/// # Notes
+///
+/// Commands must be executed/queued for execution otherwise they do nothing.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct MoveToRow(pub u16);
+
+impl Command for MoveToRow {
+    fn write_ansi(&self, f: &mut impl fmt::Write) -> fmt::Result {
+        ansi::move_to_row_csi_sequence(f, self.0)
```

Comment:
In light of #529 I would prefer that this PR also moves the ANSI code into this function such that we will not have to open an other PR in the future.

Reply:
done